data/theme/gnome-shell-sass/_endless.scss

```diff
@@ -813,7 +817,12 @@ $clubhouse-image-height: 187px;
   }
   .notification-button {
     color: white;
-    background-color: #CE5611;
+
+    // This is the default for buttons after the 3rd one (unlikely).
+    background-gradient-start: $clubhouse_button_color_d;
```

Comment:
If the start and end of the gradient is the same, then it might be better to use `background-color` here?